### PR TITLE
Remove invalid required: true in parameter schema.

### PR DESCRIPTION
**Pull request by simi**
Fixes problem with invalid specification. Not tested yet.

image

I have changed the code and regenerated the `swagger.json` file.

Part of #455 

# Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by kairoaraujo** [comment]
Rebased

**Comment by simi** [comment]
Looks good!

File: app.py [hunk 1]
```diff
@@ -7,34 +7,63 @@
...
+        version=__version__.version,
+        openapi_version=OPENAPI_VERSION,
+        description=DESCRITPTION,
+        routes=rstuf_app.routes,
+    )
+    rstuf_app.openapi_schema = openapi_schema
+    return rstuf_app.openapi_schema
+
+
+rstuf_app.openapi = _custom_openapi
```
Comment (MVrachev) [inline app.py:50]: Shouldn't we call `_custom_openapi` here?
```suggestion
rstuf_app.openapi = _custom_openapi()
```

**Review by MVrachev — COMMENTED** [review]
I need a reminder why we decided to pin a specific version of OPENAPI?
I remember there were warnings regarding our API methods, right?
Could you remind me what the problem was?

**Comment by simi** [comment]
> I need a reminder why we decided to pin a specific version of OPENAPI?

If I remember well no features of 3.1 were used and 3.1 is not supported by common tools around yet.

File: app.py (hunk 1 above)
Comment (kairoaraujo) [reply app.py:50]: No, it is correct.
https://fastapi.tiangolo.com/how-to/extending-openapi/#normal-fastapi

**Review by MVrachev — APPROVED** [review]

**Pull request merged**